### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by jdreesen**
This refactors the integration tests utilizing the [Symfony Bundle Test](https://github.com/SymfonyTest/symfony-bundle-test) library and the extension (DI) tests utilizing the [Symfony Dependency Injection Test](https://github.com/SymfonyTest/SymfonyDependencyInjectionTest) library which serve as a good basis, make them easier to read and offer some possibilities for Symfony Bundle related tests in the future.

### CODE REVIEW

**Comment by jdreesen** [comment]
Needs #71.

**Comment by mike4git** [comment]
Nice approach to keep testing configs easy to read and to handle.

What would you think about seperating model classes into two subfolders:

- Source
- Target

just to calrify what belongs to each other?!

File: tests/Populator/PersonAddressPopulatorIntegrationTest.php [hunk 1]
```diff
@@ -4,71 +4,56 @@
...
 use Neusta\ConverterBundle\Tests\Fixtures\Model\Person;
 use Neusta\ConverterBundle\Tests\Fixtures\Model\PersonAddress;
 use Neusta\ConverterBundle\Tests\Fixtures\Model\UnknownType;
 use Neusta\ConverterBundle\Tests\Fixtures\Model\User;
-use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Neusta\ConverterBundle\Tests\Support\Attribute\ConfigureContainer;
 
-class PersonAddressPopulatorIntegrationTest extends KernelTestCase
+#[ConfigureContainer(__DIR__ . '/../Fixtures/Config/person.yaml')]
+#[ConfigureContainer(__DIR__ . '/../Fixtures/Config/address.yaml')]
```
Comment (mike4git) [inline tests/Populator/PersonAddressPopulatorIntegrationTest.php:17]: Nice to see that multiple config files is easy to use

**Review by mike4git — APPROVED** [review]
Thx for this improvement of test readability

**Pull request merged**